Sources/PostgreSQL/Database/PostgreSQLDatabaseConfig.swift

```diff
@@ -7,12 +7,16 @@ public struct PostgreSQLDatabaseConfig {
     public static func `default`() -> PostgreSQLDatabaseConfig {
         return .init(hostname: "localhost", port: 5432, username: "postgres")
     }
-
-    /// Destination hostname.
-    public let hostname: String
-
-    /// Destination port.
-    public let port: Int
+	
+	public enum ServerAddress {
+		case tcp(hostname: String, port: Int)
+		case unixSocket(path: String)
+		
+		public static let `default` = ServerAddress.tcp(hostname: "localhost", port: 5432)
+		public static let defaultViaSocket = ServerAddress.unixSocket(path: "/tmp/.s.PGSQL.5432")
```

Comment:
I think `socketDefault` would be a fine name here.

Reply:
Renamed.